src/common/plugin.ts

```diff
@@ -6,6 +6,8 @@ export abstract class Plugin implements IPlugin {
 
   public abstract get name(): string;
```

Comment:
i think this property should still be referring to the plugin name. to have it refer to the command name is kind of confusing when we consider the base class and interface we're abiding to is called `Plugin`.

can we maybe refer to `name` as the name of plugin and change `displayName` to be `command` or `commandName`?